toolchain/check/convert.cpp

```diff
@@ -408,8 +408,8 @@ static auto ConvertStructToStruct(Context& context, SemIR::StructType src_type,
   }
 
   // Check that the structs are the same size.
-  // TODO: Check the field names are the same up to permutation, compute the
-  // permutation, and use it below.
+  // TODO: If not, include the name of the first source field that doesn't
+  // exist in the destination in the diagnostic.
```

Comment:
Minor, but it could also be a dest field not in the source.

Reply:
Fixed comment.